levels/challenge-questions/objectives/flatten-array/validator.js

```diff
@@ -1,19 +1,16 @@
 const assert = require("assert");
 
-const wrapInBracketsIfArray = (value) =>
-  Array.isArray(value) ? `[${value}]` : String(value);
-
 const ARRAY_ELEMENT_DIFFERENT_ERROR_MESSAGE =
   "One or more array elements are different or have different types.";
-const DEFAULT_ERROR_MESSAGE = (expected, input, testResult) => {
-  const expectedFormatted = wrapInBracketsIfArray(expected);
-  const inputFormatted = wrapInBracketsIfArray(input);
-  const testResultFormatted = wrapInBracketsIfArray(testResult);
+const DEFAULT_ERROR_MESSAGE = (expected, input, testResult, helper) => {
```

Comment:
We should probably not introduce UPPER_SNAKE_CASE functions into this code base, for consistency's sake. That's been reserver for constant and config type variables.

Reply:
Made this change!